app/pb_kits/playbook/pb_selectable_card/selectable_card.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+module Playbook
+  module PbSelectableCard
+    class SelectableCard
+      include Playbook::Props
+
+      partial "pb_selectable_card/selectable_card"
+
+      prop :checked, type: Playbook::Props::Boolean,
+                  default: false
+      prop :dark, type: Playbook::Props::Boolean,
+                  default: false
+      prop :disabled, type: Playbook::Props::Boolean,
+                  default: false
+      prop :icon, type: Playbook::Props::Boolean,
+                  default: true
+      prop :multi, type: Playbook::Props::Boolean,
+                  default: true
+      prop :name
+      prop :text
+      prop :value
+
+      def classname
+        generate_classname("pb_selectable_card_kit", dark_class, disabled_class)
+      end
+
+      def html_for
+        id.present? ? id : name
+      end
+
+      private
+
+      def dark_class
+        dark ?  "dark" : nil
+      end
+
+      def disabled_class
```

Comment:
Follow up to https://github.com/powerhome/playbook/pull/474/files?file-filters%5B%5D=.erb&file-filters%5B%5D=.js&file-filters%5B%5D=.jsx&file-filters%5B%5D=.rb#r354371129

Perhaps a more simple resolution to this as compared to the above suggestion would be simple renaming here:

```suggestion
      def enabled_disabled_class
```